packages/sdk/src/utils/math.ts

```diff
@@ -0,0 +1,3 @@
+export function min(a: bigint, b: bigint) {
+  return a < b ? a : b;
+}
```

Comment:
At least make this compatible with `Math.min` meaning that it'd support a spread of `...b` (`min(a, b, c, d, e, f)`)